templates/default/postfix-main.cf.erb

```diff
@@ -1,12 +1,5 @@
 # See /usr/share/postfix/main.cf.dist for a commented, more complete version
 
-
-# Debian specific:  Specifying a file name will cause the first
-# line of that file to be used as the name.  The Debian default
-# is /etc/mailname.
-#myorigin = /etc/mailname
-
-smtpd_banner =  ESMTP  (Ubuntu)
```

Comment:
"The text that follows the 220 status code in the SMTP greeting banner."
The default is `$myhostname ESMTP $mail_name` which is all we need.